cmd/yopass/main.go

```diff
@@ -18,6 +20,7 @@ var (
 	maxLength = pflag.Int("max-length", 10000, "max length of encrypted secret")
 	memcached = pflag.String("memcached", "localhost:11211", "Memcached address")
 	port      = pflag.Int("port", 1337, "listen port")
+	_         = pflag.Int("metrics-port", -1, "metrics server listen port")
```

Comment:
is this supposed to be underscore?

Reply:
Hehe. So I realized that all these global variables are unused, as you use viper to access the flag values. I'll change it for consistency.

Reply:
Done. It might make sense to remove all these variables at some point though to make it clear that they aren't used.

Reply:
@grobie I did not know that the variables where unused, it must have been a leftover from the time when I used `flag`. It make sense to clean them up in a separate PR. I'm going to merge this one. Thanks a lot for adding metrics to yopass ❤️